pkg/cli/cli_test.go

```diff
@@ -118,45 +118,37 @@ func (fakeService) HostService(host string) client.HostOrchestratorService {
 
 type fakeHostService struct{}
 
-func (fakeHostService) GetInfraConfig() (*apiv1.InfraConfig, error) {
-	return nil, nil
-}
+func (fakeHostService) GetInfraConfig() (*apiv1.InfraConfig, error) { return nil, nil }
```

Comment:
most of these changes are no-ops and just introduce noise

Reply:
agree, I reverted them.